docs/tutorials/start-a-private-network/alicebob.md

```diff
@@ -29,7 +29,7 @@ Alice (or whomever is playing her) should run these commands from node-template
   --chain local \
   --alice \
   --port 30333 \
-  --ws-port 9945 \
```

Comment:
I think other parts of this tutorial expect the `9945` port, e.g.
> In your web browser, navigate to https://polkadot.js.org/apps/#/settings?rpc=ws://127.0.0.1:9945.

This was changed very recently and personally I prefer it "the old way" (the way you have it here) but it needs to be consistent across the whole tutorial, of course. Also, if Alice is using `9944` I would think Bob should use `9945` and I think right now he's configured to use `9946`.

Reply:
My bad! I thought I'd accounted for all the port changes. I just corrected that and the example output to match.